qa/L0_cppunittest/test.sh

```diff
@@ -6,15 +6,15 @@ set -e
 
 # Find TE
 : ${TE_PATH:=/opt/transformerengine}
-TE_LIB_PATH=`pip3 show transformer-engine | grep Location | cut -d ' ' -f 2`
+TE_LIB_PATH=$(pip3 show transformer-engine | grep -E "Location:|Editable project location:" | tail -n 1 | awk '{print $NF}')
 export LD_LIBRARY_PATH=$TE_LIB_PATH:$LD_LIBRARY_PATH
 
 # Set parallelization parameters
 NUM_PHYSICAL_CORES=$(nproc)
 NUM_PARALLEL_JOBS=4
 
 cd $TE_PATH/tests/cpp
-cmake -GNinja -Bbuild .
+cmake -GNinja -Bbuild -DTE_LIB_PATH="$TE_LIB_PATH" .
```

Comment:
Shouldn't CMake already be checking the envvar?
https://github.com/NVIDIA/TransformerEngine/blob/aedd7e1012d4a0ad704ee9e70e4e3f91c9ba87ea/tests/cpp/CMakeLists.txt#L33